### ISSUE TITLE
syntax error problem - unexpected token

### ISSUE THREAD

**Issue by eleach** [PacktPublishing/React-and-React-Native#22]
I'm not able to run the examples. I always get  "Module build failed . . . SyntaxError . . . Unexpected token." The error always hits at the first character after:

render(
   (
     ^^^

That is, the first HTML character. Please help. I would like to run the examples. Thanks.

**Comment by adamboduch** [comment]
Hi @eleach. Are you following the steps to run the examples in the [README](https://github.com/PacktPublishing/React-and-React-Native#getting-started)? If so, can you provide me with some more details?

**Comment by eleach** [comment]
Hello Mr. Boduch,

Thanks for the reply. I think I'm OK for now. I found the answer here:


https://stackoverflow.com/questions/33460420/babel-loader-jsx-syntaxerror-unexpected-token?answertab=votes#tab-top

So, two additional steps for each code example:

1.   npm install babel-preset-react



2. Create .babelrc in root directory of project. Put in:

{
"presets": ['react']
}

------------

I'm not sure why I needed to do this. (Something specific to my setup?)

Thanks again,

Ed
